src/views/WorkspaceDetail.vue

```diff
@@ -282,6 +282,11 @@ async function update(this: any) {
   loading.value = false;
 }
 
+// handle the loading state from child emits function
+function handleLoading(newLoading: boolean) {
+  loading.value = newLoading;
+}
+
```

Comment:
I think this might conflict with the boolean value getting set elsewhere (i.e. if this sets to false, but the other is loading, we will not see the loader while it's still loading). We need 2 booleans here and the loading should be an OR over them both